### PR TITLE
nvm 1.1.0

### PR DESCRIPTION
**Pull request by Almouro**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (I ran `stepman audit --step-yml ./step.yml` instead`)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/nvm/1.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+support_url: https://github.com/almouro/bitrise-nvm-step/issues
+published_at: 2017-11-26T18:10:38.375978913+01:00
+source:
+  git: https://github.com/almouro/bitrise-nvm-step.git
+  commit: 4508f4d25172e0426d2c41b8c49a087efdae3eac
+host_os_tags:
+- osx
+- ubuntu
+type_tags:
+- npm
```
Comment (viktorbenei) [inline steps/nvm/1.1.0/step.yml]: Please change these back / normalize according to: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention

It's important for better step search / discoverability :)
Of course if you have any questions just let us know! ;)

Comment (viktorbenei) [reply steps/nvm/1.1.0/step.yml]: I sent a PR for this at https://github.com/Almouro/bitrise-nvm-step/pull/4

File: steps/nvm/1.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    description: |
+      node.js version that NVM will install.
+
+      You can use a version pointer like `6`, `6.3` or `6.3.1`.
+      You can also use the NVM version aliases like:
+      - `node`: this installs the latest version of [`node`](https://nodejs.org/en/)
+
+      More info on NVM aliases [here](https://github.com/creationix/nvm#usage)
+    is_expand: false
+    is_required: false
```
Comment (viktorbenei) [inline steps/nvm/1.1.0/step.yml]: Is there a reason why this is no longer required? Can the step work if `node_version` is not set?

Comment (Almouro) [reply steps/nvm/1.1.0/step.yml]: Updated the description in the new PR: https://github.com/bitrise-io/bitrise-steplib/pull/1278

File: steps/nvm/1.1.0/step.yml (hunk 1 above)
Comment (Almouro) [reply steps/nvm/1.1.0/step.yml]: Awesome, thanks!

**Comment by Almouro** [comment]
Made a new PR via the `share` tool: https://github.com/bitrise-io/bitrise-steplib/pull/1278

**Pull request closed without merging**